ani-cli

```diff
@@ -196,6 +205,13 @@ while [ $# -gt 0 ]; do
 	*) query="$(printf "%s" "$query $1" | sed "s/^ //;s/ /%20/g")" && shift && search="$query" ;;
 	esac
 done
+printf "\33[2K\r\033[1;34mChecking dependencies...\033[0m\n"
+dep_ch "wget" "sed" "grep" || true
+if [ "$player_function" = "download" ]; then
+	dep_ch "ffmpeg" "aria2c"
+elif ! (uname -a | grep -qE '[Aa]ndroid'); then
```

Comment:
Are you sure there wasn't a `$` in front of the parentheses?

Reply:
yes

Reply:
oh right, it's not a subshell, I'm just not used to it